src/Explorer/Panes/PanelContainerComponent.tsx

```diff
@@ -75,11 +75,17 @@ export class PanelContainerComponent extends React.Component<PanelContainerProps
   };
 
   private getPanelHeight = (): string => {
-    const consoleHeight = this.props.isConsoleExpanded
-      ? PanelContainerComponent.consoleContentHeight + PanelContainerComponent.consoleHeaderHeight
-      : PanelContainerComponent.consoleHeaderHeight;
-    const panelHeight = window.innerHeight - consoleHeight;
-    return panelHeight + "px";
+    const notificationConsole = document.getElementById("explorerNotificationConsole");
```

Comment:
If I open the save query panel first, then while the panel is open, I expand the console, the panel doesn't dynamically resize to make space for the console. I believe the old logic does but it just doesn't resize it properly.